### PR TITLE
Add container attach feature

### LINKED ISSUE #1076: Add ability to attach to a pod
**What would you like to be added**:
The ability to attach to a pod such as through `kubectl attach -it <pod-name>` for pods with tty/stdin. 

**Why is this needed**:
Feature to allow for users to attach to pods that don't have a shell or that interact with the application by attaching directly to the process in the container.

**Environment you are Lens application on:**

- Kubernetes distribution: Kubeadm / Baremetal 
- Desktop OS: Windows

### PR DESCRIPTION
**Pull request by LucaDev**
This PR adds a container/pod attach feature as described in https://github.com/lensapp/lens/issues/1076

fixes https://github.com/lensapp/lens/issues/1076

### CODE REVIEW

File: extensions/pod-menu/src/attach-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+const {
+  Util,
+} = Common;
+
+export interface PodAttachMenuProps extends Renderer.Component.KubeObjectMenuProps<Pod> {
+}
+
+export class PodAttachMenu extends React.Component<PodAttachMenuProps> {
+  async attachToPod(container?: string) {
+    Navigation.hideDetails();
```
Comment (Nokel81) [inline extensions/pod-menu/src/attach-menu.tsx]: Details should only be hidden if the rest of this function succeeds. I would recommend moving it to the bottom of this function.

Comment (LucaDev) [reply extensions/pod-menu/src/attach-menu.tsx]: I based this behavior on the shell menu. We should propably change it there too, shouldn't we? https://github.com/lensapp/lens/blob/2bc56a038303ffd9c0f2bbc5cd0d8a667317543f/extensions/pod-menu/src/shell-menu.tsx#L47-L50

Comment (Nokel81) [reply extensions/pod-menu/src/attach-menu.tsx]: Oh probably, but that should be a separate PR.

Comment (LucaDev) [reply extensions/pod-menu/src/attach-menu.tsx]: Changed it.

**Review by Nokel81 — DISMISSED** [review]

**Review by nevalla — DISMISSED** [review]

File: extensions/pod-menu/src/attach-menu.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  render() {
+    const { object, toolbar } = this.props;
+    const containers = object.getRunningContainers();
+
+    if (!containers.length) return null;
+
+    return (
+      <MenuItem onClick={Util.prevDefault(() => this.attachToPod(containers[0].name))}>
+        <Icon material="pageview" interactive={toolbar} title="Attach to Pod"/>
+        <span className="title">Attached Pod</span>
```
Comment (aleksfront) [inline extensions/pod-menu/src/attach-menu.tsx]: `Attached Pod` -> `Attach Pod`?

Comment (LucaDev) [reply extensions/pod-menu/src/attach-menu.tsx]: fixed

**Comment by aleksfront** [comment]
Also, to fix failing integration test, replace please this line https://github.com/lensapp/lens/blob/master/integration/__tests__/cluster-pages.tests.ts#L413

to 
```
await app.client.click("ul.KubeObjectMenu li.MenuItem i[title='Logs']");
```

**Comment by LucaDev** [comment]
> Also, to fix failing integration test, replace please this line https://github.com/lensapp/lens/blob/master/integration/__tests__/cluster-pages.tests.ts#L413

Seems to be working now

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**